bc/subscription/tasks.py

```diff
@@ -124,7 +128,11 @@ def process_fetch_webhook_event(fwe_pk: int):
 
 @transaction.atomic
 def make_post_for_webhook_event(
-    channel_pk: int, subscription_pk: int, fwe_pk: int, document: bytes | None
+    channel_pk: int,
+    subscription_pk: int,
+    fwe_pk: int,
+    document: bytes | None,
+    sponsor_text: str | None = None,
```

Comment:
Needs docstring.

Reply:
you're right. I'll include this parameter in the docstring.